src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -178,6 +179,10 @@ export function AccessCodeTable({
     )
   }
 
+  if (isError) {
```

Comment:
@razor-x was there a reason this wasn't included before like in the `DeviceTable`?